ocaml-lsp-server/src/ocaml_lsp_server.ml

```diff
@@ -1150,17 +1159,45 @@ let start () =
                 Server.Batch.notification batch (PublishDiagnostics d));
             Server.Batch.submit batch))
   in
-  let ocamlformat_rpc = Ocamlformat_rpc.create () in
+  let ocamlformat_rpc = Ocamlformat_rpc.create ()
+  and ocamlformat_rpc_popup = Ocamlformat_rpc.create () in
```

Comment:
So we're creating two instances of the rpc just so that we can use two different config options? That seems very wrong to me. The protocol should allow overriding config it it's really necessary.

Reply:
Yes, it seems wrong. I should have directly closed the PR when I realized it, sorry for your time taken.